microsoft-edge/extensions-chromium/store-policies/developer-policies.md

```diff
@@ -159,218 +235,327 @@ If you publish a person's Personal Information to an outside service or third-pa
 
-*   Your extension must not mine cryptocurrency.
+* Your extension must not mine cryptocurrency.
 
-### 2.9 Illegal Activity
+Extensions manipulating, cheating, or exploiting Microsoft systems, causing revenue loss, or extensions engaging in fraudulent activities, are strictly prohibited.
 
-Your extension must not contain content or functionality that encourages, facilitates or glamorizes illegal activity in the real world.
+This prohibition applies to all developers, partners, and third parties. Non-compliance may result in extension removal and account suspension or termination.
 
-### 2.10 Excessive Profanity and Inappropriate Content
+Your extension must adhere to applicable laws and regulations, Microsoft's terms of service, and applicable privacy policies. Microsoft reserves the right to review extensions for compliance and to promptly remove any noncompliant extension.
 
-*   Your extension must not contain excessive or gratuitous profanity.
 
-*   Your extension must not contain or display content that a reasonable person considers obscene.
+<!-- ------------------------------ -->
+#### 2.9 Illegal activity
 
-### 2.11 Country/Region Specific Requirements
+Your extension and the associated data must not contain content or functionality that encourages, facilitates, or glamorizes illegal activity in the real world.
 
-Content that is offensive in any country/region to which your extension is targeted is not allowed.  Content may be considered offensive in certain countries/regions because of local laws or cultural norms.  Examples of potentially offensive content in certain countries/regions include the following:
+
+<!-- ------------------------------ -->
+#### 2.10 Excessive profanity and inappropriate content
+
+* Your extension must not contain excessive or gratuitous profanity.
+
+* Your extension must not contain or display content that a reasonable person considers obscene.
+
+
+<!-- ------------------------------ -->
+#### 2.11 Country- or region-specific requirements
+
+Content that is offensive in any country or region to which your extension is targeted is not allowed.  Content may be considered offensive in certain countries or regions because of local laws or cultural norms.  Examples of potentially offensive content in certain countries or regions include the following:
 
 **China**
 
-*   Prohibited sexual content.
-*   Disputed territory or region references.
-*   Providing or enabling access to content or services that are illegal under applicable local law.
+* Prohibited sexual content.
+* Disputed territory or region references.
+* Providing or enabling access to content or services that are illegal under applicable local law.
+
 
-### 2.12 Age Ratings
+<!-- ------------------------------ -->
+#### 2.12 Age ratings
 
-#### 2.12.1 Mature Content
+<!-- todo: add content between headings if corp-level style guide applies here -->
```

Comment:
style guides say need content between headings.  but these headings are a numbered outline, so does that apply?  what could we put here between headings?

Reply:
see various examples near https://learn.microsoft.com/en-us/copilot-plugins/publish/store-policies#4-product-and-quality-guidelines

Reply:
it's an outline format; keeping as-is